spec/lib/rmt/cli/products_spec.rb

```diff
@@ -176,6 +198,22 @@
           expect(repository.mirroring_enabled).to eq(false)
         end
       end
+
+      context 'with recommended extensions' do
+        let(:product) { create :product, :with_mirrored_repositories, :with_recommended_extensions }
+        let(:extensions) { Product.recommended_extensions(product).to_a }
+
+        it 'does not disable extension repositories' do
+          product.repositories.each do |repository|
+            expect(repository.mirroring_enabled).to eq(false)
+          end
+          extensions.each do |extension|
+            extension.repositories.each do |repository|
+              expect(repository.mirroring_enabled).to eq(false)
```

Comment:
Although wait, what does this check?

Reply:
The `with_recommended_extensions` creates extensions with `with_not_mirrored_repositories` -- so mirroring is always disabled for those repos. I'm not sure what this example tries to check.